notices.txt

```diff
@@ -0,0 +1,89 @@
+=========================================================================
+==  NOTICE file corresponding to section 4 d of            	 	  ==
+==  the Apache License, Version 2.0,                                   ==
+==  in this case for NativeScript Doctor v0.9					  ==
+=========================================================================
+
+NativeScript Doctor v0.9 (the �Product�)
+
+Copyright � 2016-2018 Telerik AD. All Rights Reserved.
+
+For license information see the LICENSE.md file which accompanies this NOTICE.txt file.

```

Comment:
the file is called `notices.txt`, but here it is mentioned as `NOTICE.txt`, is this a problem?

Reply:
This is a question I also had. I will rename it to notice.txt